tests/GeneratorTest.php

```diff
@@ -89,6 +89,11 @@ public function canGenerateApiJsonFileWithChangedBaseServer()
 
         $this->assertTrue(file_exists($this->jsonDocsFile()));
 
+        $this->get(route('l5-swagger.docs'))
+            ->assertSee('https://projects.dev/api/v1')
+            ->assertDontSee('basePath')
+            ->assertStatus(200);
+
```

Comment:
Just append `->assertSee('https://projects.dev/api/v1')` after lines 98 and 103, cause now I assume we should see 2 servers in documentation files, yes?

Also, from where `https://projects.dev/api/v1'` URL is coming here?

Reply:
Yes. The PR solved the overwritten the definition of `@OA\Server` annotations. Therefore, I think the testing should be checked if the URL values were seen or not.

Reply:
`https://projects.dev/api/v1` exists in [the line of code](https://github.com/DarkaOnLine/L5-Swagger/blob/master/tests/storage/annotations/OpenApi/Anotations.php#L25)